apps/emqx_conf/include/emqx_conf.hrl

```diff
@@ -37,4 +37,61 @@
 
 -define(READONLY_KEYS, [cluster, rpc, node]).
 
+-define(CE_AUTHZ_SOURCE_SCHEMA_MODS, [
```

Comment:
this was moved  to`emqx_conf_schema_inject`, why added back ?

Reply:
ok